sdk/sdk-core/aar/src/main/java/mil/emp3/core/CoreManager.java

```diff
@@ -419,4 +423,55 @@ public void setMapGridType(IMap clientMap, MapGridTypeEnum gridType) {
         }
         mapMapping.setMapGridType(gridType);
     }
+
+    private void loadMilStdLabelLists() {
+        synchronized (CoreManager.oCommonLabels) {
+            if (CoreManager.oCommonLabels.isEmpty()) {
+                // These modifiers, if provided in the MilStd feature, must be shown
+                // along with the icon. They are the only modifiers displayed when the label setting is REQUIRED.
+                CoreManager.oRequiredLabels.add(IGeoMilSymbol.Modifier.EQUIPMENT_TYPE);
+                CoreManager.oRequiredLabels.add(IGeoMilSymbol.Modifier.SIGNATURE_EQUIPMENT);
+                CoreManager.oRequiredLabels.add(IGeoMilSymbol.Modifier.OFFSET_INDICATOR);
+                CoreManager.oRequiredLabels.add(IGeoMilSymbol.Modifier.SPECIAL_C2_HEADQUARTERS);
+                CoreManager.oRequiredLabels.add(IGeoMilSymbol.Modifier.FEINT_DUMMY_INDICATOR);
+                CoreManager.oRequiredLabels.add(IGeoMilSymbol.Modifier.INSTALLATION);
+
+                // These modifiers, if provided in the MilStd feature, are to be displayed with the icon
+                // when the label setting is COMMON.
+                CoreManager.oCommonLabels.addAll(CoreManager.oRequiredLabels);
+
+                CoreManager.oCommonLabels.add(IGeoMilSymbol.Modifier.ADDITIONAL_INFO_1);
+                CoreManager.oCommonLabels.add(IGeoMilSymbol.Modifier.ADDITIONAL_INFO_2);
+                CoreManager.oCommonLabels.add(IGeoMilSymbol.Modifier.ADDITIONAL_INFO_3);
+                CoreManager.oCommonLabels.add(IGeoMilSymbol.Modifier.HIGHER_FORMATION);
+                CoreManager.oCommonLabels.add(IGeoMilSymbol.Modifier.UNIQUE_DESIGNATOR_1);
+                CoreManager.oCommonLabels.add(IGeoMilSymbol.Modifier.UNIQUE_DESIGNATOR_2);
+            }
+        }
+    }
+
+    @Override
+    public java.util.Set<IGeoMilSymbol.Modifier> getMilStdModifierLabelList(MilStdLabelSettingEnum eLabelSetting) {
+        java.util.Set<IGeoMilSymbol.Modifier> set = null;
+
+        if (null == eLabelSetting) {
+            return null;
```

Comment:
I didn't raise an exception because the caller has a case for the value being null.